### PR TITLE
Adjust Foreword based on comments from ISO/CS

### PR DESCRIPTION
**Pull request by zygoloid**
Fix some minor discrepancies between our Foreword and the text provided by ISO/CS, and reduce the list of changes a bit.

Fixes cplusplus/nbballot#388.

### CODE REVIEW

File: source/preface.tex [hunk 1]
```diff
@@ -90,92 +80,38 @@ \chapter{Foreword}
...
 % library changes
 \item feature test macros and \libheaderref{version} header
-\item restricted valid uses of standard library functions and function templates
 \item addition of ranges and \libheaderref{ranges} header
 \item addition of calendar and time zone support
 \item addition of text formatting library and \libheaderref{format} header
-\item addition of synchronization facilities for
-waiting, notifications, semaphores, latches, and barriers, and
-\libheaderref{barrier}, \libheaderref{latch} and \libheaderref{semaphore} headers
+\item addition of \libheaderref{barrier}, \libheaderref{latch} and \libheaderref{semaphore} headers
```
Comment (jensmaurer) [inline source/preface.tex:98]: this needs an Oxford comma, I believe

**Review by jensmaurer — CHANGES_REQUESTED** [review]
one more comma, please

File: source/preface.tex (hunk 1 above)
Comment (zygoloid) [reply source/preface.tex]: Done, and squashed a little more of the more-minor content out of this list.

**Review by jensmaurer — APPROVED** [review]
looks good

**Pull request merged**